ToDoProject/src/Main.js

```diff
@@ -0,0 +1,161 @@
+import React from "react";
+import {
+  StatusBar,
+  View,
+  TextInput,
+  StyleSheet,
+  ScrollView,
+  Text,
+  AsyncStorage
+} from "react-native";
+import { styles as GlobalStyles } from "./utils/styles";
+import { Header } from "./components/Header";
+import Todos from "./components/Todos/Todos";
+const TODO_STORAGE_KEY = "TODO_STORAGE_KEY";
+
+export default class Main extends React.Component {
+  constructor(props) {
+    super(props);
+    this.loadTodos();
+    //this.deleteKeys();
+  }
+
+  state = {
+    todo: "",
+    inputError: false,
+    loading: true,
+    todos: []
+  };
+
+  loadTodos = async () => {
+    try {
+      const todos = await AsyncStorage.getItem(TODO_STORAGE_KEY);
+      this.setState({
+        todos: JSON.parse(todos) || [],
+        loading: false
+      });
+    } catch (e) {
+      console.log("Error getting Todo Items >", e);
+    }
+  };
+
+  save = async () => {
+    try {
+      AsyncStorage.setItem(TODO_STORAGE_KEY, JSON.stringify(this.state.todos));
+    } catch (e) {
+      console.log("Error while storing Todo Items >", e);
+    }
+  };
+
+  deleteKeys = async () => {
```

Comment:
Consider renaming for clarity, how about `deleteAllTodos`

Reply:
done